readthedocs/doc_builder/templates/doc_builder/conf.py.tmpl

```diff
@@ -50,12 +48,6 @@ if 'html_static_path' in globals():
 else:
     html_static_path = ['_static', '{{ static_path }}']
 
-# Add RTD Theme Path. 
-if 'html_theme_path' in globals():
-    html_theme_path.append('{{ template_path }}')
-else:
-    html_theme_path = ['_themes', '{{ template_path }}']
-
```

Comment:
Thanks for getting back to me! I was aware that this was not the most obvious place to report problems, but figured you'd be notified and you'd know best. The build works again if we intend the `html_theme = ` line so the variable is not set when running on readthedocs. Cheers!